---
title: "Sequence Shenanigans"
author: Armand Bester
date: 22 July 2025
format: revealjs
title-slide-attributes:
      data-background-image: media/Intro_DNA_mag.png
      data-background-size: contain
      data-background-opacity: "0.1"
---

[Introduction]: #####------> Section for brief introduction

## Hacking the chemistry of DNA polymerization

::: {#fig-DNA_chem layout-ncol=1}

![](media/Radford.jpg)

Radford AD, et al (2012) DOI: 10.1099/vir.0.043182-0

:::

## Clonal amplification and sequencing by synthesis

::: {#fig-clonal layout-ncol=1}

![](media/clonal_amplification.png){width=80%}

Leong IUS, et al (2014) DOI: 10.3390/medsci2020098. **a)** 454/IonTorrent, **b)** Illumina

:::

## Single molecule sequencing

::: {#fig-single_mol layout-ncol=1}

![](media/pack_bio_and_ont.webp){width=75%}

Oehler JB, et al (2023) DOI: 10.1186/s40246-023-00522-3

:::

## Oxford Nanopore Technologies

::: {#fig-ont layout-ncol=1}

![](media/MinION_GIF_08.gif)

https://nanoporetech.com/platform/technology/basecalling

:::


## Cost vs. Moore

::: {#fig-moore layout-ncol=1}

![](media/2022_Sequencing_cost_per_Human_Genome.jpg)

https://www.genome.gov/about-genomics/fact-sheets/DNA-Sequencing-Costs-Data

:::

## Phred / Q-scores

Developed by Phil Green and Brent Ewing, 1990s

$$
Q=âˆ’10.log_{10}(p)
$${#eq-Q-score}

or

```
Q10 = 90%         (1/10 chance of an incorrect base call)
Q20 = 99%         (1/100 chance of an incorrect base call)
Q30 = 99.9%       (1/1,000 chance of an incorrect base call)
Q40 = 99.99%      (1/10,000 chance of an incorrect base call)
Q50 = 99.999%     (1/100,000 chance of an incorrect base call)
Q60 = 99.9999%    (1/1,000,000 chance of an incorrect base call)
```
    These have been adopted by NGS technologies and is encoded in the FASTQ files

## Raw data and quality

::: {#fig-fastq layout-ncol=2}

![](media/fastq_fig.jpg)

![](media/ASCII-Table-wide.svg)

https://www.drive5.com/usearch/manual/fastq_files.html

:::



## The puzzel and the box

::: {#fig-puzzel layout-ncol=2}

![](media/DallE_flowers.png){width="100%"}

![](media/arious-read-types-and-read-lengths-and-the-application-of-those-reads-for-DNA-and-RNA.png)

DallE generated flowers & Read types
:::



## Infant microbiome

::: {#fig-alpha layout-ncol=1}
![](media/infant_alpha.png){width="65%"}

Alpha diversities over time
:::

$$
H_s = - \sum_{i=1}^k p_i log_2(p_i)
$${#eq-Shannon}

## ...Example results

::: {#fig-alpha layout-ncol=1}

![](media/milton.png)

* Krona reports
    
    [Krona 5b](html/krona/VRM-5B.krona.html)

    [Krona 5d](html/krona/VRM-5D.krona.html)

* [Pavian analysis](html/Uploaded_sample_set-report.html)


:::